regex/hours.regex

```diff
@@ -1 +1 @@
-([0-9]{4}[AP]M)-([0-9]{4}[AP]M)
+([0-9]{4}[AP]M)?-([0-9]{4}[AP]M)?
```

Comment:
It's a fix for issue #75 

A course like:

`NO TIMES
ENGR-092 -001	  SN	1.00	TBA	-`

Will get picked up, but it's dealt with later on in the program and displayed as an error.